### ISSUE TITLE
disable image handling

### ISSUE THREAD

**Issue by lucas-mior** [erebe/greenclip#94]
I would very much appreciate an option to disable image handling, because of the max 500kB issue.
I can't paste images into apps that are larger than this size (which occurs more often than not).
And I can't do as the guy in #90 did,
because I use xclip and it is used in another way so I can't blacklist it.
```  
maim | tee "image.png" | xclip -selection clipboard -t image/png \
```

**Comment by erebe** [comment]
Will try to add the possibility next week to do that

**Comment by erebe** [comment]
Hello,
Can you try this pre-release and let me know if it is ok for you ?
https://github.com/erebe/greenclip/releases/tag/3.4

In the config file `~/.config/greenclip.cfg` there is a new option,  `enableImageSupport` that you can set to false. It is enabled by default

**Comment by b0x4it** [comment]
I just tried it and it doesn't seem to be working. I set the following the cfg file:
enableImageSupport=False

But, it still captures png files.

**Comment by erebe** [comment]
have you killed the previous running greenclip daemon first ?

**Comment by b0x4it** [comment]
I just did and it solved the issue. Sorry for that!
However, the version still says "greenclip v3.3" which seems to be what is listed in the source code for v3.4.

**Comment by erebe** [comment]
:+1: 
Updated the code number, will do a proper release in the incoming days

**Comment by lucas-mior** [comment]
It works, thank you very much!

**Comment by erebe** [comment]
Available in release [3.4](https://github.com/erebe/greenclip/releases/tag/3.4) and update archlinux package
